packages.yml

```diff
@@ -1,3 +1,6 @@
 packages:
   - package: fivetran/zuora_source
-    version: [">=0.2.0", "<0.3.0"]
+    version: 0.2.2-a1
```

Comment:
```suggestion
    version: [">=0.2.0", "<0.3.0"]
```

Reply:
Reminder to apply after the source update is live